# this is for gui use only so the interactive markers work
string artifact_name
float64 x
float64 y
float64 z
# 1 for gui, 0 for marker server
bool sender